utils/release-candidate-testing/python/rc_test.py

```diff
@@ -0,0 +1,206 @@
+import asyncio
+from typing import List, Tuple
+import os
+import subprocess
+import sys
+
+SCRIPT_FILE = os.path.abspath(f"{__file__}/../../../cluster_manager.py")
+
+from glide import (
+    GlideClusterClient,
+    GlideClusterClientConfiguration,
+    NodeAddress,
+    GlideClient,
+    GlideClientConfiguration,
+)
+
+
+def start_servers(cluster_mode: bool, shard_count: int, replica_count: int) -> str:
```

Comment:
Can you re-use start/stop/parse from IT? No need to re-invent the wheel.

Reply:
Python is very harsh in this regard, i need the envs to be aware of each other.
If i want to reuse i need to take those functions out to the root folder, create a hack, or call it through cli.
Since it's not really something we use, it's just a fofo test, I'm not sure it's worth it.
Especially if it will affect the efficiency of the IT development because it is extra dependent.
I'm open to hearing if you think differently or you know it's easy to do so.
But importing the base on the path doesn't work.